File: Aluno.cs
public class Aluno{

    int idade;
    string nome;

    public Aluno (string nome, int idade){
        this.idade = idade;
        this.nome = nome;
    }

    public int IDADE{
        get{ return idade;}
        set{this.idade = value;}
    }

    public string NOME{
        get{return nome;}
        set{this.nome = value;}
    }   

    public string getNome(){
        return nome;
    }


}

File: Prova 2/Fila Alunos/Aluno.java
public class Aluno{
    
    public String nome;
    public int idade;
    public float nota;

    public Aluno(String nome, int idade){
        this.nome = nome;
        this.idade = idade;
        this.nota = 0;
    }

    public String getNome(){
        return nome;
    }

    public int getIdade(){
        return idade;
    }

    public void setIdade(int idade){
        this.idade = idade;
    }

    public float getNota(){
        return nota;
    }

    public void setNota(float nota){
        this.nota = nota;
    }


}
